openedx/features/caliper_tracking/transformers/course_transfomers.py

```diff
@@ -0,0 +1,47 @@
+"""
+Transformers for all the course settings related events
+"""
+
+import json
+
+
+def edx_course_home_resume_course_clicked(current_event, caliper_event):
+    """
+    This event is generated when we press the start course button on a course
+
+    :param current_event: default event log generated.
+    :param caliper_event: caliper_event log having some basic attributes.
+    :return: updated caliper_event.
+    """
+
+    object_extensions = json.loads(current_event['event'])
+
+    object_extensions.update({
+        'course_id': current_event['context']['course_id'],
+        'org_id': caliper_event['extensions']['extra_fields'].pop('org_id'),
+    })
```

Comment:
Why adding this fields in object's extensions ? Don't you think we must stay consistent in every event ? We are putting this 2 field's in main extensions.